phantombot/exports.sh

```diff
@@ -0,0 +1 @@
+export APP_PHANTOM_SERVER_PORT="25000"
```

Comment:
If this is the default port, then this entire exports.sh should be removed for simplicity